src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/Decoder.cs

```diff
@@ -10,9 +10,9 @@ namespace System.IO.Compression
 {
     internal sealed class Decoder
     {
-        internal IntPtr State = IntPtr.Zero;
-        internal BrotliDecoderResult LastDecoderResult = BrotliDecoderResult.NeedsMoreInput;
-        internal MemoryStream BufferStream;
+        internal IntPtr _state = IntPtr.Zero;
+        internal BrotliDecoderResult _lastDecoderResult = BrotliDecoderResult.NeedsMoreInput;
+        internal MemoryStream _bufferStream;
```

Comment:
Are these fields read-only outside this class? If yes, these should be modeled as read-only properties:
    internal IntPtr State { get; private set; }
If not, we need to re-visit the implementation. Fields should generally be only accessed outside the scope of a class via methods and / or properties.